actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -123,10 +123,6 @@ def _set_rendered_content_type(format) # :nodoc:
     # :api: private
     def _normalize_render(*args, &block)
       options = _normalize_args(*args, &block)
-      #TODO: remove defined? when we restore AP <=> AV dependency
-      if defined?(request) && request.variant.present?
-        options[:variant] = request.variant
-      end
       _normalize_options(options)
       options
     end
```

Comment:
this is per @rafaelfranca's comment here: https://github.com/rails/rails/pull/23663#discussion_r52836271